### ISSUE TITLE
New errors reported for existing .validaterc after upgrade to version 0.45.0

### ISSUE THREAD

**Issue by cluu12** [IBM/openapi-validator#293]
The following errors are reported in version 0.45.0 after upgrade from 0.44.0.  Is there a change that requires updating of .validaterc file?

```
[Error] Invalid configuration in .validaterc file. See below for details.

 - 'no_request_body_content' is not a valid rule for the operations category
   Valid rules are: no_request_body_name

 - 'parameters' is not a valid category.
   Valid categories are: operations, responses, schemas

 - 'no_response_codes' is not a valid rule for the responses category
   Valid rules are: no_success_response_codes, protocol_switching_and_success_code, no_response_body, ibm_status_code_guidelines
```

**Comment by rhymes** [comment]
Re-running `lint-openapi init` worked for me, or removing the fields removed in this PR https://github.com/IBM/openapi-validator/pull/286

**Comment by dpopp07** [comment]
@cluu12 - there were changes that require an update of the .validaterc file and @rhymes is correct, those are the solutions (thanks for commenting).

Unfortunately, we forgot to use our deprecation mechanism we have in the code to remove rules without errors like these. So while the file needs to be manually updated right now, this shouldn't happen again moving forward.
